src/scss/components/_header.scss

```diff
@@ -0,0 +1,95 @@
+.header {
+  position: sticky;
+  top: 0;
+  background: var(--white);
+
+  &__container {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    margin: 0 auto;
+    max-width: 1440px;
+    padding: 24px 48px;
+    width: 100%;
+  }
+
+  &__logo {
+    width: 135px;
+
+    &__image {
+      width: 100%;
+    }
+  }
+
+  &__menu {
+    align-items: center;
+    display: flex;
+    gap: 40px;
+
+    &__item .link {
+      @include link-typography;
+
+      color: var(--brand-navy);
+    }
+
+    &__item:last-child {
+      & .link {
+        @include button-typography;
+
+        align-items: center;
+        animation: 10s ease infinite button-bg-colors;
+        border-radius: 24px;
+        display: flex;
+        color: var(--white);
+        justify-content: center;
+        padding: 16px 32px;
+      }
+    }
+  }
+}
+
+@keyframes button-bg-colors {
+  0% {
```

Comment:
I think here, if between certain percentage ad other, the color will not change you can bypassed directly to the one with the change i.e (0% {background-color: var(--brand-blue)} 20% {background-color: var(--brand-green))}, please check if it works!

Reply:
I tried this method, but I don't like the result.
The configuration was intentional, it is to highlight the colors of the logo.